docs/samples/Microsoft.ML.Samples/Dynamic/SDCA.cs

```diff
@@ -38,11 +38,18 @@ public static void SDCA_BinaryClassification()
             // Read the data
             var data = reader.Read(dataFile);
 
+            // ML.NET doesn't cache data set by default. Therefore, if one reads a data set from a file and accesses it many times, it can be slow due to
+            // expensive disk operations. When the considered data can fit into memory, a solution is to cache the data in memory. Caching is especially
```

Comment:
Generally for a text sentiment dataset, I expect the runtime will be dominated by the feature engineering in the TextTransform (mostly murmur3 in the dictionary lookup, though code profiling is the final answer). I would think this would remain true with the multiple passes without caching.

```suggestion
            // expensive featurization and disk operations. When the considered data can fit into memory, a solution is to cache the data in memory. Caching is especially
```